src/SUMMARY.md

```diff
@@ -24,6 +24,7 @@
     - [Updating to 0.5](update-0.5.md)
     - [Updating to 0.6](update-0.6.md)
     - [Updating to 0.7](update-0.7.md)
+    - [Updating to 0.8](update-0.8.md)
```

Comment:
Why is the indentation different than for the other lines?

Reply:
It's not?

Reply:
Sorry, seems to be a bug with the GitHub app.